compliance/gather_licenses.py

```diff
@@ -38,32 +49,54 @@ def process_file(self, file, users):
             raise ValueError(f"Got file path that has no kind: '{file}'")
         with open(file) as attr_inp:
             if kind == "build.bazel.rules_license.license":
-                self.process_license(file, json.load(attr_inp), users)
+                self.process_license(json.load(attr_inp))
             else:
                 # TODO: When we start wrtigin other types than JSON, we have to be more careful about this
                 self.process_attribute_json(file, json.load(attr_inp), users)
 
-    def process_license(self, file, attr, users):
-        # Sample: {
+    def process_attribute_json(self, file, attr):
+        """Process a standalone attributes file."""
+        if _DEBUG > 0:
+            print(f"=== {file}")
+        kind = attr.get('kind') or None
+        if not kind:
+            raise ValueError(f"Badly formated attribute file: {file}\n{attr}")
+
+        if kind == "bazel-contrib.supply-chain.attribute.license":
+            self.process_license(attr)
+            return
+        print(f"Unknown attribute type: {kind}")
```

Comment:
No error handling nor catching of faulty licenses?

Reply:
We should fail hard. This is one of those impossible situations because it means the metadata is mal-formed.   I'll do that.

Reply:
OMG.  That was 20 minutes of trying to rebase on top of main.